packages/theme/src/cli/commands/theme/push.ts

```diff
@@ -65,7 +65,7 @@ export default class Push extends ThemeCommand {
     }),
     nodelete: Flags.boolean({
       char: 'n',
-      description: 'Runs the push command without deleting local files.',
+      description: 'Prevent deleting remote files that have been removed locally.',
```

Comment:
Adopting the same reasoning:
```suggestion
      description: 'Prevent deleting remote files that that do not exist locally.',
```